launcher/core/probeabidetector.cpp

```diff
@@ -161,6 +161,13 @@ bool ProbeABIDetector::containsQtCore(const QByteArray &line)
     // Windows Qt[X]Core[d].dll
 
     for (int index = 0; (index = line.indexOf("Qt", index)) >= 0; ++index) {
+        // Sometimes when using PySide2 on Windows we end up loading QtCore.pyd, which doesn't
+        // have information about the corresponding Qt version among its file details.
+        // We can safely skip it since that will in turn load a QtCore dll file with the right
+        // information, and that will be picked up by this function.
+        if (line.endsWith("pyd"))
```

Comment:
Can this check be done in checkQtCoreSuffix?

Reply:
Actually changed to skip pyd files altogether from `qtCoreForProcess`

Reply:
Moved back to `checkQtCoreSuffix` (although it seems a bit pessimizing), this way we can also easily test that it works through `probeabidetectortest`.